tap/line.py

```diff
@@ -23,7 +23,7 @@ def __init__(
         else:
             # The number may be an empty string so explicitly set to None.
             self._number = None
-        self._description = description
+        self._description = description.lstrip('- ')
```

Comment:
I was getting a little over eager and not fully reading the spec. I guess I will just have to remove these leading characters in post-processing.

Reply:
I remember why I removed it. I was getting `- -` to start the description because you add a `-` on [line 82](https://github.com/python-tap/tappy/pull/68/files#diff-716bf2338f5f36b821218f7ffb044093L82). I assume you would prefer to keep the current dash, but remove the one in `__str__`.